plugins/catalog/src/components/UnregisterEntityDialog/UnregisterEntityDialog.tsx

```diff
@@ -82,23 +98,36 @@ export const UnregisterEntityDialog = ({
       <DialogTitle id="responsive-dialog-title">
         Are you sure you want to unregister this entity?
       </DialogTitle>
+
       <DialogContent>
         {loading ? <Progress /> : null}
+
         {error ? (
           <Alert severity="error" style={{ wordBreak: 'break-word' }}>
-            {error.toString()}
+            {error instanceof DeniedLocationException ? (
```

Comment:
i have seen other reviews saying to prefer `name === 'DeniedLocationException' ` over `typeof` 🤷

Reply:
I changed it